### ISSUE TITLE
Sicherungsfilm 2 V H.14a*

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/xml#480]
```
Mikrofilm von 1960<!--vom April 1961 (leicht abweichend faust://bibliography/schoene1999, Bd. 2,
            S. 756),--> 
```

**Comment by gerritbruening** [comment]
vom April 1961 laut Deckblatt des Films

**Comment by dietmarpravida** [comment]
da fällt mir ein: Hast du Schönes Ausführungen im Kommentar p. 756 ad loc. noch mal nachgelesen, muss man sich in der Erl.  dazu äußern? (ich hoffe nicht)

**Comment by gerritbruening** [comment]
Ja, habe ich. Aber ich würde eine ausdrückliche Auseinandersetzung damit scheuen. Was man eher machen könnte, die Korrekturfolge km Apparateintrag auflisten.
B, S _:_ Liga _:_ B, S _wdhst G V H14b*_
welche Abkürzung wir dann noch ins Verzeichnis aufnehmen müssten.

**Comment by dietmarpravida** [comment]
(anders Albrecht Schöne in FA I 7/2, S. ###)? 

oder: 

(unzutreffend/abwegig/gottverlassen/mit bekanntem Dilettantismus, der aber zu erwarten war, wie das Amen in der Kirche/grotesk Albrecht Schöne in FA I 7/2, S. ###)? 


Was heißt B, S? 


Von: "Gerrit Brüning" <notifications@github.com> 
An: "faustedition/xml" <xml@noreply.github.com> 
CC: "dietmarpravida" <dpravida@goethehaus-frankfurt.de>, "Comment" <comment@noreply.github.com> 
Gesendet: Donnerstag, 9. August 2018 22:19:24 
Betreff: Re: [faustedition/xml] Sicherungsfilm 2 V H.14a* (#480) 



Ja, habe ich. Aber ich würde eine ausdrückliche Auseinandersetzung damit scheuen. Was man eher machen könnte, die Korrekturfolge km Apparateintrag auflisten. 
B, S : Liga : B, S wdhst G V H14b* 
welche Abkürzung wir dann noch ins Verzeichnis aufnehmen müssten.

**Comment by gerritbruening** [comment]
Siehe edit
Literaturverweise geben wir in den Erläuterungen ja fast nie. Wieso hier?

**Comment by dietmarpravida** [comment]
Na, weil das nicht so eine mittellateinische Form ist, die im Prinzip jeder kennen kann, sondern eine Äußerung zu einer Hs,* die vor uns nur er (wie auch immer) beachtet und beurteilt hat, scheint mir hier so unvermeidlich wie eine Bezugnahme auf Erich Schmidt an manchen Stellen

**Comment by dietmarpravida** [comment]
Verweis auf Ehrenzeller 1995 instar omnium im EB

**Comment by gerritbruening** [comment]
@dietmarpravida, siehe die neue Checkliste am Anfang. Kannst du bei EB ein Häkchen setzen?

**Comment by gerritbruening** [comment]
> Verweis auf Ehrenzeller 1995 instar omnium im EB

Ach so, habe ich nach dieser Angabe selbst gemacht. Wie sieht es mit den Erläuterungen zu H aus? Datierung dort so wie oben? Verweis auf Ehrenzeller oder dort nicht?

**Comment by dietmarpravida** [comment]
kann man machen, kann man vielleicht auch lassen, Die Knappheit des TranskrNachworts könnte einen Verzicht rechtfertigen, da wir sonst dort fast keine Nachweise haben

**Comment by dietmarpravida** [comment]
den in'n EB einzfügenden Text benötige ich aber, wo finde ich den?

**Comment by gerritbruening** [comment]
> kann man machen, kann man vielleicht auch lassen

Gut, dann lassen wir es, aber ist die Datierung so wie am Kopf angegeben?

**Comment by dietmarpravida** [comment]
ja, das ist noch zu korrigieren, da derzeit alle beide Daten im Text stehen. Ich würde (zumindest im EB) auch "vor 1981" angeben (man muss aber Landeck deswegen wohl nicht zitieren, hätte aber seinen Charme, dann wäre die Famiie beinahe komplett)

**Comment by dietmarpravida** [comment]
Ich habe Ehrenzeller und Landeck jetzt doch beide im Transkriptionsnachwort eingetragen. Die dortige bibliographische Angabe scheint mir in der Form nötig.

**Comment by gerritbruening** [comment]
Gut, um den XML-Rest kümmere ich mich.

**Comment by thvitt** [comment]
Hier müsste auch eine Makrogenese-Regel 2 H –temp-pre→ 2 V H.14a* ergänzt werden.

**Comment by dietmarpravida** [comment]
cf.  #670
